app-games/src/main/java/com/aptoide/android/aptoidegames/appview/AppViewScreen.kt

```diff
@@ -670,12 +674,30 @@ fun AppRating(
         .size(12.dp),
     )
     Text(
+      modifier = modifier
+        .padding(end = 8.dp),
       text = if (rating.avgRating == 0.0) "--"
       else TextFormatter.formatDecimal(rating.avgRating),
       maxLines = 1,
       style = AGTypography.InputsS,
       overflow = TextOverflow.Ellipsis,
     )
+    downloads?.let {
+      Text(
+        text = "${formatDownloads(it)} downloads",
+        maxLines = 1,
+        style = AGTypography.InputsS,
+        overflow = TextOverflow.Ellipsis,
+      )
+    }
+  }
+}
+
+fun formatDownloads(downloads: Int): String {
```

Comment:
This could be in the `TextFormatter` that is present in the aptoide_ui module. It also should account for larger values like billions. It can be similar to the function `withSuffix` that is in the `TextFormatter`